examples/_includes/templates/list.xml.njk

```diff
@@ -23,7 +23,7 @@ hv_list_tag: "root"
         {% set item_style = item_style ~ " item-last" %}
       {% endif %}
       <item
-        href="{{ screen.data.permalink }}"
+        href="/hyperview/public/{{ screen.data.permalink }}"
```

Comment:
all the changes inside examples/ folder is to prepend the path with /hyperview/public/